examples/workflows/upstream/volumes-existing.py

```diff
@@ -0,0 +1,29 @@
+from hera.workflows import Container, Workflow, Steps, Step
+from hera.workflows import models as m
+
+with Workflow(
+    generate_name="volumes-existing-",
+    entrypoint="volumes-existing-example",
+    volumes=[m.Volume(name="workdir", persistent_volume_claim={"claim_name": "my-existing-volume"})],
+) as w:
+    with Steps(name="volumes-existing-example") as s:
+        Step(name="generate", template="append-to-accesslog")
+        Step(name="print", template="print-accesslog")
```

Comment:
Can we use the new callable syntax here?

The template order does not matter so we can define our template first and then reference them there.